Mathlib/Data/Matrix/Rank.lean

```diff
@@ -274,4 +275,34 @@ theorem rank_eq_finrank_span_row [LinearOrderedField R] [Finite m] (A : Matrix m
   rw [← rank_transpose, rank_eq_finrank_span_cols, transpose_transpose]
 #align matrix.rank_eq_finrank_span_row Matrix.rank_eq_finrank_span_row
 
+section MulNonsingInv
+
+/-! ### Lemmas about pre or post multiplying by an invertible matrix. -/
+
+/-- Pre (Left) multiplying by an invertible matrix does not change the rank -/
+lemma rank_mul_isUnit [DecidableEq n] [CommRing R]
+    (A: Matrix n n R) (B: Matrix m n R) (hA: IsUnit A.det) :
```

Comment:
```suggestion
    (A : Matrix n n R) (B : Matrix m n R) (hA : IsUnit A.det) :
```
and below.

Reply:
I've unresolved this as it still applies

Reply:
I've unresolved this as it still applies

Reply:
It seems I didn't fetch the changes in my local copy and when I merged last they where undone